src/standardized wip/IAR_LU_linear.py

```diff
@@ -39,6 +39,10 @@ def __init__(self, bvalues=None, thresholds=None, bounds=None, initial_guess=Non
         super(IAR_LU_linear, self).__init__(bvalues, thresholds, bounds, initial_guess)
         
         # Check the inputs
+        self.osipi_check_required_bvalues()
```

Comment:
I like where you're going with this, but adding these checks everywhere is cumbersome. I'm not sure exacly the best approach, but a few come to mind.
1. Adding them to the base class. This would abstract it away, but also make it less flexible when the values are set.
2. Creating setters for these values, and in the setters we can call these methods.

I'm open to other suggestions too. Maybe @IvanARashid has some ideas here?